### DISCUSSION TITLE
Add universal -trace flag to print every line of V source right before it executes

### DISCUSSION THREAD

**Discussion by elimisteve** [vlang/v#11650] [Ideas]
...just like `bash -x myprogram.sh`! Which is super useful; much more pleasant than manually adding `print` statements all over the place, or having to use a debugger.

I see [verbose_trace](https://github.com/vlang/v/blob/master/cmd/tools/modules/scripting/scripting.v#L16-L21) in `cmd/tools/modules/scripting/scripting.v`, so perhaps something similar could be used to implement this?

**Comment by spytheman** [comment]
Unless we get a V interpreter, I think that will be hard to do in a general way.

**Comment by elimisteve** [comment]
It's fine if we first need to compile with -debug to build a binary that lets us -trace, which surely makes this much more feasible, yes @spytheman ?
